app/models/emoji.rb

```diff
@@ -190,6 +190,7 @@ def self.load_custom
             e.name = emoji.name
             e.url = emoji.upload&.url
             e.group = emoji.group || DEFAULT_GROUP
+            e.created_by = User.find(emoji.user_id).username # not sure if we want to do this same thing below when the skip_db global setting is on?
```

Comment:
this is my only outstanding question 🤔

Reply:
> not sure if we want to do this same thing below when the skip_db global setting is on?

I don't think so because the emojis below are plugin emojis that come from the `register_emoji` API, which doesn't take a user id argument, so there's no concept of a user-created plugin emoji.